tests/custom_hypothesis_support.py

```diff
@@ -19,6 +19,14 @@
 ])
 primitive_values = primitive_types.flatmap(lambda s: s())
 
+non_bool_primitive_types = hs.sampled_from([
```

Comment:
You don't need these. Just use the already-defined primitive types in the new test.